mobile/examples/java/hello_world/MainActivity.java

```diff
@@ -61,6 +63,7 @@ protected void onCreate(Bundle savedInstanceState) {
                    Log.d("MainActivity", "Envoy async internal setup completed");
                    return null;
                  })
+                 // .enablePlatformCertificatesValidation(true)
```

Comment:
was this commented out on purpose?

Reply:
Actually https requests interleaving with http request doesn't work neither with nor without platform cert validator. But running them alone worked. I don't know the reason yet. I will split out this change and let the rest get in first.